docs/build-parachains.md

```diff
@@ -1,42 +1,32 @@
 ---
```

Comment:
Joe, not sure why the final PR has the same file removed, and this one added. Here it is in a diff view for you anyhow.